source/compliance/tsc/index.rst

```diff
@@ -0,0 +1,60 @@
+
+   Providing security to IT products and systems is a key element in improving an entity's overall risk management strategy and internal control structure. These criteria focus on the logical and physical protection of information, systems, and networks. The common criteria (CC) are organized into nine subsections, which are:
+
+   -  CC1: Control environment
+   -  CC2: Communication and Information
+   -  CC3: Risk Management and Design and Implementation of Controls
+   -  CC4: Control Activities
+   -  CC5: Monitoring of Controls
+   -  CC6: Logical and Physical Access Controls
+   -  CC7: System Operations
+   -  CC8: Change Management
+   -  CC9: Risk Mitigation
+
+   In summary, SOC 2, TSC, COSO, and CC are all frameworks and standards organizations use to manage risks and ensure the effectiveness of their internal controls. SOC 2 and TSC focus on the security, availability, processing integrity, confidentiality, and privacy of an organization's systems and services. COSO provides a broader framework for enterprise risk management and internal control, while CC offers a method for evaluating the security features of IT products and systems. Organizations may use a combination of these frameworks and standards to develop a comprehensive risk management and compliance strategy.
+
+-  **TSC additional criteria**
+
+   The TSC additional criteria are an extension of the common criteria. Organizations can use these additional criteria to address precise security requirements not defined by the conventional TSC common criteria.
+
+   Wazuh assists with these criteria by performing log collection, file integrity monitoring, configuration assessment, threat detection, vulnerability assessment, and automated threat response.
+
+This document outlines use cases that show how Wazuh helps users comply with the TSC common criteria, and the additional criteria for availability, confidentiality, and processing integrity. We have also created the `Using Wazuh for TSC 2017 requirements guide <https://documentation.wazuh.com/resources/using-wazuh-for-TSC-2017-requirements-guide.pdf>`__, which complements this document. Please refer to the guide for more details on how Wazuh helps meet TSC requirements.
+
+The following sections outline some of the technical requirements that Wazuh supports:
+
+.. toctree::
+   :caption: The COSO Principle and Common Criteria
+   :maxdepth: 1
+
+   common-criteria/cc2.1
```

Comment:
Define specific texts for the side menu as the section titles are too long. 

![image](https://github.com/wazuh/wazuh-documentation/assets/61882981/7add2942-b3a9-47d2-ba2a-d5d154bbf706)